### PR TITLE
Problem (fix #325): integration test failure

### LINKED ISSUE #325: Problem: integration test failure
https://github.com/crypto-com/chain-main/pull/324#issuecomment-752357066

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- add more timeout

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-com/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-com/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-com/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-com/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Comment by tomtau** [comment]
it seems to still fail: https://github.com/crypto-com/chain-main/pull/326/checks?check_run_id=1628740446#step:5:478

File: integration_tests/utils.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from pystarport.ports import rpc_port
 
 
-def wait_for_block(cli, height, timeout=60):
+def wait_for_block(cli, height, timeout=6000):
```
Comment (tomtau) [inline integration_tests/utils.py:15]: is the timeout in seconds? if so, 100 minutes may be too high?

**Review by tomtau — COMMENTED** [review]
looks all right

File: integration_tests/utils.py (hunk 1 above)
Comment (yihuang) [reply integration_tests/utils.py]: yes, changed to 120seconds now, should be enough.

**Comment by yihuang** [comment]
I think the main issue is in github action execution environment, when the load becomes high, some processes seems hangs for a very long time. We have seen this happens when run ledger tests together with other tests in parallel before. This time we change the test worker number from 7 to 3, should make it more stable than before.

**Review by tomtau — APPROVED** [review]

**Pull request merged**